### PR TITLE
Create v-rising.md

### PR DESCRIPTION
**Pull request by git-st0m**
hey, just added a V Rising review on MacBook M3

### CODE REVIEW

**Comment by JoshuaBrest** [comment]
Please fix all lint issues.

File: src/game-support/v-rising.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+V Rising
```
Comment (JoshuaBrest) [inline src/game-support/v-rising.md:1]: Add heading
```diff
- V Rising
+ # V Rising
```

File: src/game-support/v-rising.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
```
Comment (JoshuaBrest) [inline src/game-support/v-rising.md:2]: Add aliases.
```diff
+ <!-- script:Aliases [] -->
  
```

File: src/game-support/v-rising.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+{{#template ../templates/rating.md status=Platinum installs=Yes opens=Yes}}
+
+{{#template ../templates/steam.md id=1604030}}
```
Comment (JoshuaBrest) [inline src/game-support/v-rising.md:5]: After making the above changes, run `./script/generate.mjs`.

**Review by JoshuaBrest — CHANGES_REQUESTED** [review]

**Review by hahayupgit — APPROVED** [review]

**Pull request merged**